### PR TITLE
[REG-503] Update default timeouts

### PR DESCRIPTION
**Pull request by nAmKcAz**
- Pro tip, be careful on PR for the upstream branch.. 
- Change default timeouts to something more reasonable for RGBot

### CODE REVIEW

**Comment by nAmKcAz** [comment]
@vontell These 2 changes seem to hide the issues I was seeing.  Now.. do we publish this python package to make it easy for users to access ?

File: src/javascript/events.py [hunk 1]
```diff
@@ -169,15 +169,21 @@ def loop(self):
...
+                    cbid = inbound["cb"] if "cb" in inbound else None
+                    if "c" in inbound and inbound["c"] == "pyi":
+                        j = inbound
+                        self.callbackExecutor.add_job(r, cbid, self.pyi.inbound, inbound)
+                    if r in self.requests:
+                        lock, timeout = self.requests[r]
+                        # Give the main event loop here up to 60 seconds before deciding the connection died
+                        # Many RGBot actions like pathfinding could take much longer than the original value of 5 seconds
+                        # Once one of these calls fails, the IPC is dead
+                        barrier = threading.Barrier(2, timeout=60)
```
Comment (vontell) [inline src/javascript/events.py:183]: Is 60 enough time for all awaits? I'm a little worried about things like someone using a function that awaits for something that may never even happen.

I see your try catch now at least stops the fatal error - so even if the timeout is reached, can the user detect that and try again?

**Review by vontell — CHANGES_REQUESTED** [review]

**Comment by vontell** [comment]
> @vontell These 2 changes seem to hide the issues I was seeing. Now.. do we publish this python package to make it easy for users to access ?

Yup! We can publish our own and pip install that package. I think we should also make an issue / ticket in the original repo and see if that person can make this fix as well, maybe they can make the timeout configurable.

**Review by abeizer — APPROVED** [review]

File: src/javascript/events.py (hunk 1 above)
Comment (nAmKcAz) [reply src/javascript/events.py:183]: Yes, 60 is gracious plenty.  If a user has a single call into rgbot taking over 60 seconds, they're doing it wrong I think.  That call would be holding up the event loop and any other calls until it finished.

**Pull request merged**